docs/extensions.adoc

```diff
@@ -1,3 +1,4 @@
+[#_extensions_chapter]
```

Comment:
Can't you use the autogenerated id? https://asciidoctor.org/docs/user-manual/#auto-generated-ids
Otherwise we use the `[[id]]` style elsewhere

Reply:
@p2w34 What do you think about the Leonard comment?

Reply:
@p2w34 Assuming your are still here, what do you think about the Leonard comment?

Reply:
@szpak Apologies, I must have missed your last comment from a year ago. This is a very minor issue which lasts so long. As so much time have passed please simply address the issue the way you think appropriate - I'd refrain myself from comment to not to prolong it even longer.

Reply:
Actually this issue is long fixed by merging my per #1122 that supersedes this PR as was also discussed in that PR, you just didn't close this one here @szpak